app/helpers/Evaluation/EvaluationStatus/IEvaluable.php

```diff
@@ -14,29 +14,29 @@ interface IEvaluable
      * Query if the evaluation is ready
      * @return boolean The result
      */
-    function hasEvaluation(): bool;
+    public function hasEvaluation(): bool;
```

Comment:
The `public` seems a bit redundant... is there any special reason for it?

Reply:
It is required by PSR-12 validator.